README.md

```diff
@@ -0,0 +1,75 @@
+# clay-sitemaps
+
+A suite a tools for generating sitemaps for Clay sites.
+
+Clay-sitemaps provides two "standard" sitemap middleware, one for text and one for XML, that can get you up and running quickly. It also provides utilities that can help you create your own sitemap logic.
+
+## Quick Start
+
+```
+const sitemaps = require('clay-sitemaps')(someAmphoraInstance);
+
+app
+  .get('/sitemap.txt', sitemaps.standardText())
+  .get('/sitemap.xml', sitemaps.standardXML({
+    engines: { // engines for rendering sitemap templates
+      handlebars: yourHandlebarsEngine
```

Comment:
For our own setup, we need to be able to either read the engine from Amphora or pass it in like this because `nymag-handlebars` creates a new instance of `handlebars` and then `sites` does some more to it. If `clay-sitemaps` just does `require('handlebars')`, we don't get a handlebars instance with the same configuration/helpers.